src/components/PersonCard/PersonCard.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  Box,
+  Card,
+  CardActions,
+  CardContent,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import clsx from 'clsx';
+import { FC } from 'react';
+import * as React from 'react';
+import { Picture } from '../Picture';
+import { Link } from '../Routing';
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
+  personCard: {
+    maxWidth: '247px',
+    paddingTop: spacing(1),
+  },
+  cardContent: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  personName: {
+    paddingBottom: spacing(0.5),
+  },
+  personImage: {
+    width: '86px',
+    height: '86px',
+    paddingBottom: spacing(2),
+  },
+  roleName: {
+    marginBottom: spacing(1),
+  },
+  organizationName: {
+    marginBottom: spacing(0.25),
+  },
+  statusCircle: {
+    height: '8px',
+    width: '8px',
+    marginRight: spacing(1),
+    borderRadius: '50%',
+    display: 'inline-block',
+  },
+  cardActions: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    padding: spacing(2),
+    border: '0.5px solid #e0e0e0',
+  },
+  activeColor: {
+    backgroundColor: palette.primary.main,
```

Comment:
Let's hardcode `#619A55` here. We always think of "active" as green, regardless of our theme colors.